scheduler/src/cook/components.clj

```diff
@@ -404,10 +402,15 @@
                                                     :progress-sample-interval-ms (* 1000 60 5)}
                            default-uri-config {:cache true
                                                :executable true
-                                               :extract false}]
+                                               :extract false}
+                           executor-uri-configured (and (:uri executor) (not (str/blank? (get-in executor [:uri :value]))))]
+                       (when executor-uri-configured
+                         (log/warn "Executor uri value is missing, the uri config will be disabled" {:executor executor}))
                        (cond-> (merge default-executor-config executor)
-                               (:uri executor)
-                               (update :uri #(merge default-uri-config %1)))))))
+                               executor-uri-configured
+                               (update :uri #(merge default-uri-config %1))
+                               (not executor-uri-configured)
+                               (dissoc :uri))))))
```

Comment:
Seems like this would be cleaner as an `if executor-uri-configured`

Reply:
Done.